transifex/common/console.py

```diff
@@ -58,3 +62,26 @@ def prompt(prompt_msg, description=None, default=None, new_line=False, vtype=Non
             description=description))
 
     return click.prompt(prompt_msg, default=default, type=(vtype or str))
+
+
+def pluralized(one, other, cnt_value):
+    """Render an ICU pluralized string with the given parameters.
+
+    :param unicode one: the string for singular
+    :param unicode other: the string for plural
+    :param int cnt_value: the value of the plural counter variable
+    :return: a rendered string that has taken into account the given
+    :rtype: unicode
+    """
+    icu_string = '{cnt, plural, one {[one]} other {[other]}}'\
+        .replace('[one]', one)\
+        .replace('[other]', other)
+
+    return StringRenderer.render(
```

Comment:
I believe it's best to stick with `StringRenderer`. This is the only renderer we have and ICU is just the syntax we base rendering on. If we have to introduce more advanced business logic in the future and ICU is just a (small) part of that, maybe it would make sense to rename that to `ICUStringRenderer` and have a more generic class that deals with the high level rendering. Until then I think we are good.

Reply:
@dbendilas I am good with that thanks for answering that.